modules/cnv-updating-access-mode-to-enable-live-migration.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-to-enable-live-migration_{context}"]
+= Updating access mode to enable LiveMigration
+
+Storage class defaults in {VirtProductName} are configured to provide ReadWriteOnce (RWO) access mode
+by default. Therefore, LiveMigration is not enabled by default unless you
+alter the access mode of your virtual machine.
+
+.Prerequisites
+* In order to use LiveMigration, virtual machines must have a persistent volume claim (PVC) with a shared
+ReadWriteMany (RWX) access mode to be live migrated.
+
+.Procedure
+To override the default RWO access mode, update the storage class configuration
+by doing the following:
+
+. Export the storage class object.
++
+----
+$ export STORAGE_CLASS="nfs"
```

Comment:
This command is not exporting the object, rather it is exporting (to the shell environment) the 'STORAGE_CLASS' variable. In the context of this procedure, this step is unnecessary in an example like this because it is simpler just to use the value, 'nfs', in the following 'oc patch' command.

Reply:
Removed the export step.